.eslintrc

```diff
@@ -0,0 +1,31 @@
+root: true
+
+env:
+  node: true
+  browser: true
+  mocha: true
+
+rules:
+  curly: [2, "multi-line"]
```

Comment:
One of the advantage of YAML is that it's human-friendly (more than human readable) :-)
One benefit of this is that you do not need to wrap strings with quotes, could you remove them please? I have tested without and it works fine, so it's nicer!